java/test/org/openqa/selenium/InvalidSelectorTest.java

```diff
@@ -0,0 +1,33 @@
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.openqa.selenium;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.testing.JupiterTestBase;
+
+class InvalidSelectorTest extends JupiterTestBase {
+
+  @Test
+  void testOldPage() {
+    driver.get(pages.simpleTestPage);
+    assertThatExceptionOfType(InvalidSelectorException.class).isThrownBy(driver.findElement(By.xpath("invalid-xpath"))::click);
+  }
+}
```

Comment:
This test is not needed. You need to check the `ElementFindingTest` class, and fix the failing tests, please.

Reply:
@diemol done, and reran all browser tests on 'chrome' and 'firefox' to confirm.
Thanks a lot for your support!
![image](https://user-images.githubusercontent.com/19201898/222649501-bb3e2968-c306-4902-abcc-ded84dc35d90.png)

Reply:
Great!